ecolex/settings.py

```diff
@@ -37,6 +37,13 @@
 # Selenium
 #TEST_RUNNER = 'django_selenium.selenium_runner.SeleniumTestRunner'
 
+# URLs used for legislation import
+DOC_URL = os.environ.get('DOC_URL')
+HTML_URL = os.environ.get('HTML_URL')
+PDF_URL = os.environ.get('PDF_URL')
+KEYWORDS_URL='http://extwprlegs1.fao.org/cgi-bin/xml.exe?database=valid&' \
```

Comment:
This can also be taken from `env`. You can add the current values as defaults so that the project works without defining them, but also overriding them is easy.